app/views/annotation_categories/_annotation_text.html.erb

```diff
@@ -66,7 +66,7 @@
     <% end %>
     <% if annotation_text[:annotation_category].nil? %>
       <div class='annotation-text-details'>
-        <%= t('activerecord.models.submission', {'count': 1}) + ': ' %>
+        <%= t('activerecord.models.submission', **{'count': 1}) + ': ' %>
```

Comment:
it's better to do `t('activerecord.models.submission', count: 1)` to remove the hash entirely